docs/layouts/_default/index.json.json

```diff
@@ -0,0 +1,11 @@
+{{- $index := slice -}}
```

Comment:
Woah, why is GitHub marking all of this is red?

Reply:
Maybe  because the  json template is `.json` but  isn't itself valid json?